services/schedulable_transaction_body.proto

```diff
@@ -293,16 +293,16 @@ message SchedulableTransactionBody {
     /**
      * Transaction body for a scheduled transaction to create a new node.
      */
-    com.hedera.hapi.node.addressbook.NodeCreateTransactionBody nodeCreate = 42;
```

Comment:
Please do not break the correct packaging (both here and below).

Reply:
Updated.